src/vos/vos_gc.c

```diff
@@ -226,6 +226,16 @@ gc_drain_cont(struct vos_gc *gc, struct vos_pool *pool,
 	return gc_drain_btr(gc, pool, &cont->cd_obj_root, credits, empty);
 }
 
+static int
+gc_free_cont(struct vos_gc *gc, struct vos_pool *pool, struct vos_gc_item *item)
+{
+	vos_dtx_table_destroy(&pool->vp_umm,
```

Comment:
Roughly how many PMDK operations could be executed in vos_dtx_table_destroy()? I'm afraid that could result in a huge PMDK transaction when there are too many elements in the dtx table.

Reply:
The main operation for vos_dtx_table_destroy() is to release the vos_dtx_scm_blob list. We do NOT touch the DTX entries in the vos_dtx_scm_blob(). So there will be at most thousands of vos_dtx_scm_blob. I do not think it is burden for PMDK transaction.